gitlabform/gitlabform/test/test_push_rules.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from gitlabform.gitlabform import GitLabForm
+from gitlabform.gitlabform.test import create_group, create_project_in_group, get_gitlab, create_readme_in_project, \
+    GROUP_NAME
+
+PROJECT_NAME = 'push_project'
+GROUP_AND_PROJECT_NAME = GROUP_NAME + '/' + PROJECT_NAME
+
+
+@pytest.fixture(scope="module")
+def gitlab(request):
+    gl = get_gitlab()
+
+    create_group(GROUP_NAME)
+    create_project_in_group(GROUP_NAME, PROJECT_NAME)
+    gl = get_gitlab()
+
+    def fin():
+        gl.delete_project(GROUP_AND_PROJECT_NAME)
+
+    request.addfinalizer(fin)
+    return gl  # provide fixture value
+
+
+set_project_push_rules = """
```

Comment:
Can you please also add a test to change existing push rules? Being unable to change existing ones was what made you implement this PR anyway, @grzesuav . :)

See an example here: https://github.com/egnyte/gitlabform/blob/master/gitlabform/gitlabform/test/test_group_members.py#L193

Reply:
I tried to follow that, but failed.

Can you briefly look a this ?

Reply:
Sure. I’ll let you know.

Reply:
I created a PR to your fork & branch with the fixes here: https://github.com/grzesuav/gitlabform/pull/1/files

Reply:
Nvm, I was able to push to your branch directly.